### PR TITLE
Release note for 2.0.11 vsts

### PR DESCRIPTION
**Pull request by Yiming-Jia**

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.11 - March, 24 2020
+* Updated [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Do we include documentation updates as a part of release notes? Typically, if something requires an update to the version of the extension, then we consider it to be a change in that release and include it in release notes. Is that the case here?

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 2]
```diff
@@ -4,9 +4,15 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
...
+  * Added `FAQ` section.
+  * Removed `Preview` tag.
```
Comment (Yiming-Jia) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: * put on the separate graph.
* Azure App Configuration pipeline task for Azure DevOps is now generally available.
* Added `FAQ` section to the [documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task).

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 3]
```diff
@@ -4,9 +4,14 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.11 - March, 24 2020
+* Azure App Configuration pipeline task for Azure DevOps is now generally available.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: This needs to start with "The"

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Added

**Pull request merged**